QC/wdl/workflows/snv_indel_assembly.wdl

```diff
@@ -0,0 +1,190 @@
+version 1.0
+
+import "../tasks/deepvariant.wdl" as runDeepVariant
```

Comment:
> import

😠

Reply:
(Don't actually remove the imports, I will concede that in this scenario it lends itself to better organization.)

Reply:
LOL.
Yeah, in this case we are allowing people to access the DeepVariant and PMDV standalone WDLs which I think is very helpful.